library/ssl_cli.c

```diff
@@ -156,6 +156,15 @@ static int ssl_write_hostname_ext( mbedtls_ssl_context *ssl,
 
     return( 0 );
 }
+
+int mbedtls_ssl_write_hostname_ext( mbedtls_ssl_context *ssl,
```

Comment:
```suggestion
int mbedtls_ssl_tls13_write_hostname_ext( mbedtls_ssl_context *ssl,
```

Reply:
This is not TLS 1.3 specific thus this is ok I think to not have `_tls13_` in the function name. Otherwise, no need to wrap `ssl_write_hostname_ext`, just change its name to `mbedtls_ssl_write_hostname_ext`.